pyproject.toml

```diff
@@ -32,8 +32,8 @@ classifiers = [
 ]
 dependencies = [
     "nbformat",
-    "mdit_py_plugins",
-    "markdown-it-py>=1.0.0",
+    "mdit-py-plugins",
+    "markdown-it-py>=2.0",
```

Comment:
I am planning to release this PR in `jupytext_1.16.1` so I'd rather let this change of dependencies on `markdown-it-py` for `jupytext_1.17.0`.

Reply:
Ok, let me add the CI to run for `1.0` in this case then revert the commit after that

Reply:
Great idea! Thanks